.github/CODEOWNERS

```diff
@@ -1,2 +1,3 @@
 # Default
 * @fission-codes/development
+* @jeckhart
```

Comment:
@jeckhart, since we're updating things, can you add me here: @zeeshanlakhani. I'll talk to @walkah and Fission to see how we want to org this moving forward.

Reply:
I'm flexible. I don't think we have any plans to shut down the org (right @bmann ?) - but if you want to set `CODEOWNERS` to be individuals or if anyone wants to be removed from the team so they don't get pinged. I'm happy to help sort that out.

Reply:
@jeckhart ah ok. Yeah, we can update the codeowners separately in another clean-up.